chapter_recurrent-modern/seq2seq.md

```diff
@@ -74,9 +102,20 @@ output, state = encoder(X)
 output.shape
 ```
 
+```{.python .input}
+#@tab pytorch
+encoder = Seq2SeqEncoder(vocab_size=10, embed_size=8, num_hiddens=16,
+                         num_layers=2)
+encoder.eval()
+X = torch.zeros(4, 7).long()
```

Comment:
```suggestion
X = d2l.zeros((4, 7), dtype=torch.long)
```
We are now making a few functions standard across all frameworks. See [here](https://github.com/d2l-ai/d2l-en/blob/28b0d4ce61c3f6f79e95f57d0be284c71453f9f4/d2l/torch.py#L35)